File: 2356-largest-combination-with-bitwise-and-greater-than-zero/2356-largest-combination-with-bitwise-and-greater-than-zero.cpp
class Solution {
public:
    int largestCombination(vector<int>& candidates) {
        vector<int> bitCnt(24, 0);
        for(int i=0; i<24; i++){
            for(int num : candidates){
                if((num & (1<<i)) != 0) bitCnt[i]++;
            }
        }
        return *max_element(bitCnt.begin(), bitCnt.end());
    }
};

File: 2356-largest-combination-with-bitwise-and-greater-than-zero/2356-largest-combination-with-bitwise-and-greater-than-zero.java
class Solution {
    public int largestCombination(int[] candidates) {
        int ans = 0;
        for(int i=0; i<24; i++){
            int cnt = 0;
            for(int x : candidates){
                if((x & (1<<i)) != 0) cnt++;
            }
            ans = Math.max(ans, cnt);
        }
        return ans;
    }
}